examples/src/main/java/com/expediagroup/sdk/rapid/examples/salesprofiles/RapidPartnerSalesProfile.java

```diff
@@ -0,0 +1,14 @@
+package com.expediagroup.sdk.rapid.examples.salesprofiles;
+
+
+public abstract class RapidPartnerSalesProfile {
```

Comment:
Should we add a comment on the value of the profiles here?

Reply:
Yes, definitely, I'm planning to do this after I have a conversation with Elvin to finalize the naming and come up with a proper definition to what it represents.